packages/app/src/messages/actions/toggle-thread-read-status.action.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import { toggleThreadReadStatusRequest } from "App/messages/requests"
+import { MessagesEvent } from "App/messages/constants"
+import { ToggleThreadReadStatusError } from "App/messages/errors"
+import { Thread } from "App/messages/reducers/messages.interface"
+
+export const toggleThreadReadStatus = createAsyncThunk<
```

Comment:
```suggestion
export const toggleThreadsReadStatus = createAsyncThunk<
```

Reply:
Fixed in [74a6af8](https://github.com/mudita/mudita-center/pull/728/commits/74a6af816f864d38862a81f9539521a7f38d27b0)